src/infra/core/Cargo.toml

```diff
@@ -121,6 +121,7 @@ tokio-util = { version = "0.7", default-features = false, features = [
 tracing = "0.1"
 url = { version = "2", features = ["serde"] }
 walkdir = "2"
+console = "0.15.8"
```

Comment:
`console` should not appear in such a deep layer, it's supposed to work in CLI code only, while `infra` is a shared code for between CLI and API server.